components/joomla/joomla.app.mjs

```diff
@@ -121,19 +121,23 @@ export default {
       }
       return data;
     },
-    async listUsers($) {
-      const generator = this.paginate({
+    async listUsers({
+      $, context = {},
```

Comment:
I wouldn't recommend passing `context` object because it will depend only on `async options` property functions. what you can do is to pass a property like `url` so you can check if that property is not `undefined` then you can use it when calling `makeRequest`